### PR TITLE
Update documentation and resolve ped issues

### LINKED ISSUE #199: [PE-D] Adding repeat recipes
Not sure if this is intended, but the app allows adding multiples of the same recipe with slight variations such as a different image or different tag.

For example,

addR n/salad i/lettuce, carrots, olive oil c/40 instr/Prepare the ingredients. Toss the ingredients together. Serve. img/https://www.onceuponachef.com/images/2019/07/Big-Italian-Salad.jpg t/yummy 

and 

addR n/salad i/lettuce, carrots, olive oil c/40 instr/Prepare the ingredients. Toss the ingredients together. Serve. img/https://www.onceuponachef.com/images/2019/07/Big-Italian-Salad.jpg t/yummy t/healthy

results in both recipes being added, despite both being the same recipe, just with an extra tag.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: PrestonTYR/ped#1

### LINKED ISSUE #201: [PE-D] Adding multiple same ingredients to the fridge
When I add the same ingredients to the fridge, instead of adding together, the ingredients remain seperate.

For example, 
if I do addF i/tomato -1kg, 
followed by addF i/tomato -2kg,

When I use the fridge command to view the list of ingredients, the ingredient, tomato does not stack to become tomato 3kg.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: PrestonTYR/ped#3

### LINKED ISSUE #208: [PE-D] Possible feature bug: if I add a duplicate ingredient into the fridge, shouldn't it just accumulate in the fridge?
As per title. E.g. if I want to add the same thing twice, it should jus accumulate I think. 

current fridge:

new addition: 

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: rtshkmr/ped#1

### LINKED ISSUE #209: [PE-D] Duplicating the same ingedient doesn't accumulate ingredients?
I think repeat ingredients should cause an accumulation. but even if quantity is not determined by the parsing, there's an extra comma here. Still an aesthetic bug maybe? 

input: 

`editR 8 n/test3 i/foo i/foo c/40 instr/ make img/images/default.jpg t/xx`

outcome:

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: rtshkmr/ped#2

### LINKED ISSUE #221: [PE-D] Duplicate issue not detected when images are different
Steps to reproduce:
1. addR n/salad i/lettuce, tomato, olive oil c/40 instr/Cook pasta. Serve immediately. t/fast t/easy
2. addR n/salad i/lettuce, tomato, olive oil c/40 img/images/healthy1.jpg instr/Cook pasta. Serve immediately. t/fast t/easy
You can also change it to any *invalid image like healthy10.jpg

It is the same recipe except there no image in the first entry.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: todoge/ped#2

### LINKED ISSUE #225: [PE-D] Possible unintended search
The searchR function also finds matching keywords. If you searchR n/a, all recipes containing "a" appears. It does not seem to be reflected in the UG where it seems to be match by the word present in the recipe. Not if the contains the character.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: todoge/ped#6

### LINKED ISSUE #226: [PE-D] Duplicate Fridge ingredients
Small changes in the name and quantity allows same ingredients to be added.

-------------
Labels: `severity.High` `type.FunctionalityBug`
original: todoge/ped#7

### LINKED ISSUE #227: [PE-D] Quantity not adding
If I buy more of an ingredient, the quantity should add up. Otherwise, I might have alot of duplicated ingredients with different quantities.

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: todoge/ped#8

### LINKED ISSUE #236: Make sure no duplicate ingredients in addR (check ordering)

### PR DESCRIPTION
**Pull request by caitlinjee**

### CODE REVIEW

File: src/main/java/seedu/address/model/recipe/Recipe.java [hunk 1]
```diff
@@ -212,9 +217,10 @@ public String toString() {
         final StringBuilder builder = new StringBuilder();
         builder.append(getName())
                 .append(" Ingredient: ")
-                .append(ingredients.stream()
+                /*.append(ingredients.stream()
                         .map(item -> item.getQuantity() + " " + item.getValue())
-                        .reduce("", (a, b) -> b.equals("") ? a : b + ", " + a))
+                        .reduce("", (a, b) -> b.equals("") ? a : b + ", " + a))*/
```
Comment (TianYong-Goh) [inline src/main/java/seedu/address/model/recipe/Recipe.java]: Is it forgot to remove this comment part?

File: src/main/java/seedu/address/ui/RecipeCard.java [hunk 2]
```diff
@@ -84,9 +84,12 @@ public RecipeCard(Recipe recipe, int displayedIndex) {
         };
         getRoot().widthProperty().addListener(stageSizeListener);
 
-        ingredients.setText(recipe.getIngredient().stream()
+        /*ingredients.setText(recipe.getIngredient().stream()
                 .map(item -> item.getQuantity() + " " + item.getValue())
-                .reduce("", (a, b) -> b.equals("") ? a : b + ", " + a));
+                .reduce("", (a, b) -> b.equals("") ? a : b + ", " + a));*/
```
Comment (TianYong-Goh) [inline src/main/java/seedu/address/ui/RecipeCard.java]: Same as above

File: src/main/java/seedu/address/ui/SingleRecipeCard.java [hunk 3]
```diff
@@ -68,9 +68,12 @@ public SingleRecipeCard(Recipe recipe) {
         };
         getRoot().widthProperty().addListener(stageSizeListener);
 
-        ingredients.setText(recipe.getIngredient().stream()
+        /*ingredients.setText(recipe.getIngredient().stream()
                 .map(item -> item.getQuantity() + " " + item.getValue())
-                .reduce("", (a, b) -> b.equals("") ? a : b + ", " + a));
+                .reduce("", (a, b) -> b.equals("") ? a : b + ", " + a));*/
```
Comment (TianYong-Goh) [inline src/main/java/seedu/address/ui/SingleRecipeCard.java]: Same as above

**Review by TianYong-Goh — APPROVED** [review]

**Review by daongochieu2810 — APPROVED** [review]

**Review by jiaqi20 — APPROVED** [review]
lgtm

**Review by oliviajohansen — APPROVED** [review]
lgtm

**Pull request merged**